src/Notifications/TwoStepVerificationNotification.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Notifications;
+
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class TwoStepVerificationNotification extends Notification
+{
+    use Queueable;
+
+    public function __construct(
+        protected User $user,
+        protected int $code,
+    ) {}
+
+    public function via(): array
+    {
+        return ["mail"];
```

Comment:
I would prefer `NotificationChannels::MAIL` here. Of course there's no class like that, but you could create it and adjust other notification classes too.

Reply:
done